src/components/Subset.jsx

```diff
@@ -58,8 +58,8 @@ export const SubsetPreview = ({subset}) => {
     // FIXME: show title in selected language, not just first in the name array.
     // TODO: show subset in other languages - switch button for language?
 
-    const from = subset.validFrom?.toISOString().substr(0, 10);
-    const to = subset.validUntil?.toISOString().substr(0, 10);
+    const from = subset.validFrom instanceof Date ? subset.validFrom.toISOString().substr(0, 10) : subset.validFrom;
+    const to = subset.validUntil instanceof Date ? subset.validUntil.toISOString().substr(0, 10) : subset.validUntil;
```

Comment:
This change and the similar one in SubsetCodes was made to handle the fact that input type=date passes strings, and not Dates. This will let the site fail softly no matter if validFrom and validUntil are Dates or something else, but that's not necessarily a good thing. Now that I made the input type=date elements parse their strings into Date objects, we should consider to just leave the original lines in place instead, they would work now like before.

Reply:
Reasonable. 
My mentor once said to me, if I would need to check the date type, then it could be a sign of a deeper problem. 
Nice that you've found a proper solution for Dates in the AppContext. 
Restore the originals then, please.

Reply:
alright, will do!